src/storage/src/hummock/iterator/concat_inner.rs

```diff
@@ -132,8 +80,9 @@ impl<TI: SstableIteratorType> ConcatIteratorInner<TI> {
                 old_iter.collect_local_statistic(&mut self.stats);
             }
         } else {
-            let table = self.tables[idx]
-                .prefetch(&self.sstable_store, &mut self.stats)
+            let table = self
+                .sstable_store
+                .sstable(&self.tables[idx], &mut self.stats)
```

Comment:
random thought: we can prefetch the next sstable meta similar to the block prefetch in sstable iterator. not sure how much improvement we can get though

Reply:
> random thought: we can prefetch the next sstable meta similar to the block prefetch in sstable iterator. not sure how much improvement we can get though

We can get more memory :rolling_on_the_floor_laughing:

Reply:
> > random thought: we can prefetch the next sstable meta similar to the block prefetch in sstable iterator. not sure how much improvement we can get though
> 
> We can get more memory :rolling_on_the_floor_laughing:

True. But it looks acceptable because the memory usage is proportional to number of levels, not number of SSTs, and we have very few levels.

Reply:
But after we support meta-cache refill after compaction, the meta-cache miss rate is very low.
So it is not necessary to prefetch the next TableHolder for delete-range (